prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -193,6 +196,9 @@ def filter(
             for finding in findings:
                 # We don't send findings to not enabled regions
                 if finding.Resources[0].Region not in findings_per_region:
+                    logger.error(
```

Comment:
I think this can make a lot of noise, can we log just once per not enabled region?

Reply:
What if I change it to WARNING?